src/core/src/op/squared_difference.cpp

```diff
@@ -20,3 +45,31 @@ std::shared_ptr<ov::Node> ov::op::v0::SquaredDifference::clone_with_new_inputs(c
     check_new_args_count(this, new_args);
     return std::make_shared<ov::op::v0::SquaredDifference>(new_args.at(0), new_args.at(1), this->get_autob());
 }
+
+bool ov::op::v0::SquaredDifference::evaluate(TensorVector& outputs, const TensorVector& inputs) const {
+    OV_OP_SCOPE(v0_SquaredDifference_evaluate);
+    OPENVINO_ASSERT(outputs.size() == 1);
+
+    outputs[0].set_shape(infer_broadcast_shape(this, inputs));
+    using namespace ov::element;
+    return IF_TYPE_OF(v0_SquaredDifference_evaluate,
+                      OV_PP_ET_LIST(f32),
+                      squared_difference::Evaluate,
+                      inputs[0].get_element_type(),
+                      inputs[0],
+                      inputs[1],
+                      outputs[0],
+                      inputs[0].get_shape(),
+                      inputs[1].get_shape(),
+                      get_autob());
+}
+
+bool ov::op::v0::SquaredDifference::has_evaluate() const {
+    OV_OP_SCOPE(v0_SquaredDifference_has_evaluate);
+    switch (get_input_element_type(0)) {
+    case element::f32:
```

Comment:
minor:
can we just check the lib bin size before/after adding f32, f16,i16,i32 etc types here?
It's just interesting point